pytket/conanfile.py

```diff
@@ -38,7 +38,7 @@ def requirements(self):
         self.requires("pybind11_json/0.2.14")
         self.requires("symengine/0.13.0")
         self.requires("tkassert/0.3.4@tket/stable")
-        self.requires("tket/1.3.49@tket/stable")
+        self.requires("tket/1.3.50@tket/stable")
```

Comment:
This shouldn't need updating.

Reply:
I've reverted - let's see if the same issue comes up